test3.rb

```diff
@@ -0,0 +1,22 @@
+def solution(a, b, c)
+  word_arr = [[*("a" * a).chars], [*("b" * b).chars], [*("c" * c).chars]]
+  new_word = ""
```

Comment:
Always use single quotes unless you need interpolation.
```suggestion
  new_word = ''
```